Rakefile

```diff
@@ -58,7 +58,17 @@ namespace :generate do
 
   desc "generate a bunch of stuff with generators"
   task :stuff do
-    in_example_app "bin/rake rails:template LOCATION='../../example_app_generator/generate_stuff.rb'"
+    in_example_app "bin/rake #{template_task_name} LOCATION='../../example_app_generator/generate_stuff.rb'"
+  end
+end
+
+def template_task_name
+  gem 'rails'
+
+  if Gem.loaded_specs['rails'].version >= Gem::Version.create('5.0')
```

Comment:
You should just be able to check the rails version from it's constant, we try to avoid using rubygems where possible.